openslides_backend/action/actions/motion_comment/create_delete_update.py

```diff
@@ -1,12 +1,76 @@
+from typing import Any, Dict
+
 from ....models.models import MotionComment
+from ....permissions.permissions import OrganisationManagementLevel, Permissions
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import PermissionDenied
+from ....shared.patterns import Collection, FullQualifiedId
+from ...action import Action
 from ...action_set import ActionSet
+from ...generics.delete import DeleteAction
+from ...generics.update import UpdateAction
 from ...mixins.create_action_with_inferred_meeting import (
-    get_create_action_with_inferred_meeting,
+    CreateActionWithInferredMeeting,
 )
 from ...util.default_schema import DefaultSchema
 from ...util.register import register_action_set
 
 
+class PermissionMixin(Action):
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        super().check_permissions(instance)
+
+        # check for superuser
+        if self.user_id > 0:
+            user = self.datastore.get(
+                FullQualifiedId(Collection("user"), self.user_id),
+                [
+                    "organisation_management_level",
+                ],
+            )
+            if (
+                user.get("organisation_management_level")
+                == OrganisationManagementLevel.SUPERADMIN
+            ):
+                return
```

Comment:
I think you missunderstood the description: The **meeting admin** is implicitely in all write groups, not the **superadmin**. The latter has nothing to do with meeting interna. So instead of checking the organization level, you should fetch the field `used_as_admin_group_in_meeting_id` from all groups of the request user and check this way if he is admin.

Reply:
Check if user is in the meeting admin group. Remove the superuser check.